react/components/EXPERIMENTAL_Table/constants.ts

```diff
@@ -40,6 +60,10 @@ export const FIELDS_BOX_ITEM_HEIGHT = 36
 export const TABLE_HEADER_HEIGHT = 36
 export const EMPTY_STATE_SIZE_IN_ROWS = 5
 export const DEFAULT_SCROLLBAR_WIDTH = 17
+export const NESTED_ROW_PREFIX_WIDTH = 36
+export const BULK_ACTIONS_HEIGHT = 56
+export const BULK_ACTIONS_TRANSITION =
+  'height 0.2s ease-in-out, padding 0.2s ease-in-out'
```

Comment:
You could use the `grow` class  from `Tachyons`.

Reply:
Not really